include/swift/AST/DiagnosticsParse.def

```diff
@@ -1504,6 +1504,8 @@ ERROR(attr_differentiable_expected_label,none,
 // differentiating
 ERROR(attr_differentiating_expected_original_name,PointsToFirstBadToken,
       "expected an original function name", ())
+ERROR(attr_differentiating_expected_linear_wrt,none,
+"expected either 'linear' or 'wrt:'", ())
```

Comment:
```suggestion
ERROR(attr_differentiating_expected_label_linear_or_wrt,none,
      "expected either 'linear' or 'wrt:'", ())
```

Reply:
I don't believe this suggestion was applied before the code was merged.

Reply:
Oh I thought this was a name change of the variable. Before it was `attr_differentiating_expected_linear_wrt` but now I changed it to `attr_differentiating_expected_label_linear_or_wrt`

Reply:
https://github.com/apple/swift/pull/25257/commits/d787aef95070dcb9f9c9688bc8e4d1b60d702cca

Reply:
It's also about the indentation of the second line.

Reply:
Oops will fix this in the new PR misinterpreted the suggestion